filament/src/PerViewUniforms.cpp

```diff
@@ -69,6 +69,7 @@ void PerViewUniforms::prepareCamera(const CameraInfo& camera) noexcept {
     mat4f const& viewFromWorld = camera.view;
     mat4f const& worldFromView = camera.model;
     mat4f const& clipFromView  = camera.projection;
+    const bool isPerspective = clipFromView[2][3] == -1;
```

Comment:
nit: Existing code checks `p[3][2] != 0` to determine projection type ([Camera.cpp#L235](https://github.com/google/filament/blob/afd657d6af0dc7ead4a083685a6e13815e0a9902/filament/src/Camera.cpp#L235)), I'd prefer staying consistent with that.

Reply:
e12ac5e85118f0f266f4f52161ee6720ae4b6b26